packages/common/src/workflow-options.ts

```diff
@@ -39,13 +39,57 @@ export enum WorkflowIdReusePolicy {
 
   /**
    * Terminate the current workflow if one is already running.
+   *
+   * This option belongs in `WorkflowIdConflictPolicy` but it is here for backwards compatibility.
+   * With no Running Workflow, the behavior is the same as `WORKFLOW_ID_REUSE_POLICY_ALLOW_DUPLICATE`,
+   * but with a Running Workflow, it behaves like `WORKFLOW_ID_CONFLICT_POLICY_TERMINATE_EXISTING`.
+   *
+   * *Note: When setting this option, `WorkflowIdConflictPolicy` should have value `WORKFLOW_ID_CONFLICT_POLICY_UNSPECIFIED`*
    */
   WORKFLOW_ID_REUSE_POLICY_TERMINATE_IF_RUNNING = 4,
```

Comment:
I had a go at making this more direct (I used the word "deprecated" since I don't think people should be using this in new code, right?)

```suggestion
  /**
   * Terminate the current workflow if one is already running, otherwise allow a duplicate ID.
   *
   * Deprecated: use `WORKFLOW_ID_CONFLICT_POLICY_TERMINATE_EXISTING` and `WORKFLOW_ID_REUSE_POLICY_ALLOW_DUPLICATE` instead.
   *
   * This option belongs in `WorkflowIdConflictPolicy` but it is here for backwards compatibility.
   * *Note: If setting this option, `WorkflowIdConflictPolicy` must be left unspecified.*
   */
  WORKFLOW_ID_REUSE_POLICY_TERMINATE_IF_RUNNING = 4,
```